components/Stats/index.jsx

```diff
@@ -13,6 +13,7 @@ export default ({
   countries
 }) => {
   const router = useRouter();
+  const activeCases = confirmed.value - deaths.value - recovered.value
```

Comment:
`const activeCases = confirmed?.value - deaths?.value + recovered?.value`

Reply:
Actually if one of them fail (undefined), the whole calculation will result in a NaN.

Reply:
Untrue: https://covid-19-next-git-fork-taoufik-ajaanit-master.smakosh.now.sh/AS

Reply:
Because it's passed to `NumberFormat`!
Actually my remark is true! try it in the console

In my 13" laptop with an edited zoom level by the OS, the active cases text is wrapped
 
![image](https://user-images.githubusercontent.com/8569885/77258336-4c4ac580-6c7a-11ea-9b34-c48adb143c9c.png)

Reply:
yup, that component handles that case, Hey Taoufik, can you minimize the `gap` to `1` or `2`?